openfisca_france/model/prestations/minima_sociaux/cmu.py

```diff
@@ -275,7 +276,7 @@ def function(self, simulation, period):
 
         res = cmu_br_i_par[CHEF] + cmu_br_i_par[PART] + forfait_logement
 
-        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca)
+        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca + paje_prepare)
```

Comment:
Thanks, i'll call you tomorrow

Florian

2015-05-06 17:47 GMT+02:00 Mahdi Ben Jelloul <notifications@github.com>:

> In openfisca_france/model/prestations/minima_sociaux/cmu.py
> <https://github.com/openfisca/openfisca-france/pull/244#discussion_r29771359>
> :
>
> > @@ -275,7 +276,7 @@ def function(self, simulation, period):
> >
> >          res = cmu_br_i_par[CHEF] + cmu_br_i_par[PART] + forfait_logement
> >
> > -        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca)
> > +        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca + paje_prepare)
>
> Yep whenever you want
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openfisca/openfisca-france/pull/244/files#r29771359>.
>